api/breaker/breaker.go

```diff
@@ -103,7 +103,7 @@ func (s State) String() string {
 
 // ErrStateTripped will be returned from executions performed while the CircuitBreaker
 // is in StateTripped
-var ErrStateTripped = trace.ConnectionProblem(nil, "breaker is tripped")
+var ErrStateTripped = trace.ConnectionProblem(nil, "Circuit Breaker is tripped. Unable to connect to Teleport Auth service.")
```

Comment:
changed it to api service

Reply:
While this advice is sound for internal errors, user-facing messages should still be proper sentences with correct punctuation.

In fact, `trace` errors like `ConnectionProblem` intentionally store the underlying error separate from the user-facing message:

```
// ConnectionProblem returns new instance of ConnectionProblemError
func ConnectionProblem(err error, message string, args ...interface{}) Error {
	return newTrace(&ConnectionProblemError{
		Message: fmt.Sprintf(message, args...),
		Err:     err,
	})
}
```

Reply:
Please see update which I think is appropriate error whichever client is connecting:

```
Unable to reach Teleport API on Auth Service. Number of attempts has tripped circuit breaker.
```

Reply:
I've made a more generic message for now.  So this error will get tripped in a function like this in `lib/client/client.go`. It seems like this point we could provide more context that it can't reach the auth service. Unless there's a easier way to do it more generically. 

```go
func (c *Client) GetResources(ctx context.Context, req *proto.ListResourcesRequest) (*proto.ListResourcesResponse, error) {
	if err := req.CheckAndSetDefaults(); err != nil {
		return nil, trace.Wrap(err)
	}

	resp, err := c.grpc.ListResources(ctx, req)
	return resp, trace.Wrap(err)
}
```

```
Stack Trace:
        github.com/gravitational/teleport/api@v0.0.0/client/client.go:3117 github.com/gravitational/teleport/api/client.(*Client).GetResources
        github.com/gravitational/teleport/api@v0.0.0/client/client.go:3249 github.com/gravitational/teleport/api/client.GetResourcePage[...]
        github.com/gravitational/teleport/api@v0.0.0/client/client.go:3316 github.com/gravitational/teleport/api/client.GetAllResources[...]
        github.com/gravitational/teleport/lib/client/api.go:2278 github.com/gravitational/teleport/lib/client.(*TeleportClient).ListNodesWithFilters
        github.com/gravitational/teleport/tool/tsh/common/tsh.go:2165 github.com/gravitational/teleport/tool/tsh/common.onListNodes.func1
        github.com/gravitational/teleport/lib/client/api.go:570 github.com/gravitational/teleport/lib/client.RetryWithRelogin
        github.com/gravitational/teleport/tool/tsh/common/tsh.go:2164 github.com/gravitational/teleport/tool/tsh/common.onListNodes
        github.com/gravitational/teleport/tool/tsh/common/tsh.go:1318 github.com/gravitational/teleport/tool/tsh/common.Run
        github.com/gravitational/teleport/tool/tsh/common/tsh.go:544 github.com/gravitational/teleport/tool/tsh/common.Main
        github.com/gravitational/teleport/tool/tsh/main.go:24 main.main
        runtime/proc.go:267 runtime.main
        runtime/asm_arm64.s:1197 runtime.goexit
```

Reply:
Yes I didn't want leave it inaccurate as specific to Auth Service. Did you have any thoughts on where we should add context?